app/facades/websocket/WebSocketController.scala

```diff
@@ -251,6 +299,28 @@ class WebSocketController @Inject() (actorSystem:  ActorSystem,
    */
   private def getDsId(request: RequestHeader) = request.queryString("dsId").head
 
+  /**
+    * Extracts `salt` from the requests query string
+    *
+    * @param request
+    * @return
+    */
+  private def getSalt(request: RequestHeader) = request.queryString("salt").head
```

Comment:
Will it properly handle the exception resulted from "salt" being missing from the url?

Reply:
Actualy, getSalt() methods never used. I will remove it.